apps/spruce/src/components/Spawn/spawnHostModal/getFormSchema.tsx

```diff
@@ -178,6 +180,9 @@ export const getFormSchema = ({
                     defineSetupScriptCheckbox: {
                       enum: [true],
                     },
+                    warning: {
```

Comment:
nit. could be nice to indicate what the item is, but not necessary just a suggestion
```suggestion
                    warningBanner: {
```